manifests/jenkins-agent.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: jenkins-agent
+parameters:
+  - description: Secret name containing any additional root CA certificates
+    name: ADDITIONAL_ROOT_CA_CERTS_SECRET
+    value: additional-root-ca-certs
+objects:
+  - apiVersion: v1
+    kind: ImageStream
+    metadata:
+      name: jenkins-agent
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      name: jenkins-agent
+    spec:
+      source:
+        dockerfile: |
+          FROM overridden
+          COPY certs/* /etc/pki/ca-trust/source/anchors/
+          RUN update-ca-trust
+        secrets:
+          - destinationDir: certs
+            secret: additional-root-ca-certs
```

Comment:
I think this needs to be changed to:
```yaml
        secrets:
          - secret:
              name: additional-root-ca-certs
            destinationDir: certs
```

Reply:
Yes, looking at https://docs.openshift.com/container-platform/4.10/rest_api/workloads_apis/buildconfig-build-openshift-io-v1.html#spec-source-secrets, you're right. Fixed!